modules/ROOT/pages/6.5-release-notes.adoc

```diff
@@ -47,6 +47,15 @@ This resulted in an inconsistent end-user experience.
 
 {productname} now presents end-users with uniform merge tag highlighting across all supported browsers.
 
+=== Insert merge tag button and menu item was not disabled when selection was inside a non-editable context.
```

Comment:
```suggestion
=== *Insert → Merge tag* menu item, and equivalent toolbar button was not disabled when selection was inside an element with a `contenteditable="false"` attribute set
```
1. spelling out element with a contenteditable="false" attribute in full for correctness’s sake.
2. copy edits to UX and UI item spelling and presentation to exactly match presentation in the editor.